### ISSUE TITLE
Standardize files names

### ISSUE THREAD

**Issue by LorenaSoledad** [raml-org/raml-tck#103]
In order to keep consistency of tests, it might be great if we could standardize the input and output filenames for each test. Besides, we can divide those tests containing two tests in one folder into two separate folders. For example, the following folder: `tests/raml-1.0/Annotations/test001` contains four files:

- apiValid.raml
- apiValid-tck.json
- apiInvalid.raml
- apiInvalid-tck.json

But we can separate the valid/invalid tests into two different folders/tests.

For standardizing filenames, one suggestion could be:
- "input.raml" for the main RAML file.
- "api-tck.json" for the expected output
- Any other file will be ignored (except in includes, etc.)

(Or any other input/output format that suits best for you...)

Please, feel free to suggest anything for this issue.
Thanks.

**Comment by sichvoge** [comment]
Hi @LorenaSoledad. It think its a brilliant idea to standardize file names and make the maintenance easier in the future. 

Wold you have two different folders for invalid and valid tests, or do you think its enough to add some extensions to the file name?

**Comment by jayS-de** [comment]
It would be good if there is only one entry point with the same name in each folder. So valid and invalid raml files should reside in separate folders. They could be inside the test folder itself like:

```
tests/raml-1.0/Annotations/test001/valid/input.raml
tests/raml-1.0/Annotations/test001/valid/api-tck.raml
tests/raml-1.0/Annotations/test001/invalid/input.raml
tests/raml-1.0/Annotations/test001/invalid/api-tck.raml
```

What should be kept in mind are the overlays and extension. The overlay tests use a api.raml and a NewOverlay.raml

The JS parser is called using the api.raml and given the overlays as options. The java parser is called using the Overlay file. So for all parsers the way of handling overlays and extension should also be unified in the TCK. My proposal would be to say use the overlay file as the entry point. So instead of:

```
api.raml
NewOverlay.raml
```

use

```
api.raml
input.raml (formerly known as NewOverlay.raml)
```

**Comment by sichvoge** [comment]
Hi @jayS-de. You are everywhere :D

Have a look into the `wip/2.0.0` branch. It's a rewrite from the original TCK version. I am still trying to add more content, though ;)

**Comment by jstoiko** [comment]
this is addressed in upcoming v2.0. closing.
